spec/statement.dd

```diff
@@ -1372,14 +1372,14 @@ with (expression)
 
 --------------
 {
-    Object tmp;
-    tmp = expression;
+    Object* tmp;
+    tmp = &expression;
     ...
     tmp.ident;
 }
 --------------
 
-        $(P Note that $(EXPRESSION) only gets evaluated once.
+        $(P Note that $(EXPRESSION) only gets evaluated once and not copied by-value.
```

Comment:
We should improve this. The address taking example is not edifying because it doesn't work with rvalues. Furthermore, there's no obvious lowering. I think it's enough to leave the example as is and simply mention in the text the following:

`$(P Note that $(EXPRESSION) only gets evaluated once and is not copied.`